<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QMainWindow" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Ponies</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="addpage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2" rowspan="3" colspan="3">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="7" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="addpony_button">
             <property name="text">
              <string>Add pony</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>80</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_ponyname">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="image_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="QTreeView" name="available_list">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTabBar" name="tabbar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="usesScrollButtons" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="activepage">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QPushButton" name="removepony_button">
             <property name="text">
              <string>Remove pony</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTreeView" name="active_list">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="configpage">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" colspan="7">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>&amp;General</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="1" column="1" colspan="2">
             <widget class="QCheckBox" name="alwaysontop">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="3">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="toolTip">
               <string>Toggles if ponies are always on top of other windows</string>
              </property>
              <property name="text">
               <string>&amp;Always on top</string>
              </property>
              <property name="buddy">
               <cstring>alwaysontop</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_bypass_wm">
              <property name="toolTip">
               <string>Bypass the X11 window manager, showing ponies on every desktop on top of every window</string>
              </property>
              <property name="text">
               <string>Show on every &amp;virtual desktop</string>
              </property>
              <property name="buddy">
               <cstring>x11_bypass_wm</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QCheckBox" name="x11_bypass_wm">
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="toolTip">
               <string>Specifies in which directory the pony data and configuration are</string>
              </property>
              <property name="text">
               <string>Pony data di&amp;rectory</string>
              </property>
              <property name="buddy">
               <cstring>ponydata_directory</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="ponydata_directory">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QToolButton" name="change_ponydata_dir">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="toolTip">
               <string>Toggle if interactions are to be executed</string>
              </property>
              <property name="text">
               <string>&amp;Interactions enabled</string>
              </property>
              <property name="buddy">
               <cstring>interactions_enabled</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QCheckBox" name="interactions_enabled">
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="3">
             <widget class="QGroupBox" name="box_advanced">
              <property name="title">
               <string>Advanced options</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="toolTip">
                  <string>Toggle if debug messages are printed to console</string>
                 </property>
                 <property name="text">
                  <string>&amp;Debug messages</string>
                 </property>
                 <property name="buddy">
                  <cstring>debug_enabled</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="debug_enabled">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="show_debuglog">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Show debug log</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>S&amp;peech</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="2" column="1">
             <widget class="QSpinBox" name="textdelay">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="suffix">
               <string> ms</string>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>2000</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="toolTip">
               <string>For how long the speech is to stay on screen</string>
              </property>
              <property name="text">
               <string>&amp;Text delay</string>
              </property>
              <property name="buddy">
               <cstring>textdelay</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="toolTip">
               <string>Toggles if ponies are to speak</string>
              </property>
              <property name="text">
               <string>&amp;Speech enabled</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="buddy">
               <cstring>speechenabled</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="speechenabled">
              <property name="toolTip">
               <string/>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="toolTip">
               <string>NOT ACTIVE. Toggles the playing of sounds when a pony speaks.</string>
              </property>
              <property name="text">
               <string>Play s&amp;ounds</string>
              </property>
              <property name="buddy">
               <cstring>playsounds</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="playsounds">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Sound</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="toolTip">
               <string>How frequently ponies speak</string>
              </property>
              <property name="text">
               <string>Sp&amp;eech probability</string>
              </property>
              <property name="buddy">
               <cstring>speechprobability</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="speechprobability">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string> %</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QPushButton" name="acceptsettings">
          <property name="text">
           <string>Accept</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="resetsettings">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="show_advanced">
          <property name="text">
           <string>&amp;Show advanced options</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>qtabbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>speechenabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>textdelay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speechenabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>playsounds</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speechenabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addpony_button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>add_pony()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removepony_button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>remove_pony_activelist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetsettings</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>load_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptsettings</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>save_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playsounds</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_ponydata_dir</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>change_ponydata_directory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_advanced</sender>
   <signal>toggled(bool)</signal>
   <receiver>box_advanced</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_debuglog</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>show_debuglog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save_settings()</slot>
  <slot>remove_pony()</slot>
  <slot>remove_pony_all()</slot>
  <slot>remove_pony_activelist()</slot>
  <slot>newpony_list_changed(QModelIndex)</slot>
  <slot>add_pony()</slot>
  <slot>update_active_list()</slot>
  <slot>toggle_window(QSystemTrayIcon::ActivationReason)</slot>
  <slot>load_settings()</slot>
  <slot>lettertab_changed(int)</slot>
  <slot>change_ponydata_directory()</slot>
  <slot>show_debuglog()</slot>
 </slots>
</ui>
